transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/extensions/MessageExtension.kt

```diff
@@ -92,17 +93,22 @@ private fun List<AttachmentContent>.toAttachments(): Map<String, Attachment> {
 private fun List<StructuredMessage.Content>.toQuickReplies(): List<ButtonResponse> {
     val filteredQuickReply = this.filterIsInstance<QuickReplyContent>()
     val filteredButtonResponse = this.filterIsInstance<ButtonResponseContent>()
-    return when {
-        filteredQuickReply.isNotEmpty() -> filteredQuickReply.map {
-            it.quickReply.run { ButtonResponse(text, payload, "QuickReply") }
-        }
 
-        filteredButtonResponse.isNotEmpty() -> filteredButtonResponse.map {
-            it.buttonResponse.run { ButtonResponse(text, payload, type) }
+    if (filteredQuickReply.isNotEmpty()) {
+        return filteredQuickReply.map {
+            it.quickReply.run { ButtonResponse(text, payload, "QuickReply") }
         }
+    }
 
-        else -> emptyList()
+    if (filteredButtonResponse.isNotEmpty()) {
+        return filteredButtonResponse
+            .mapNotNull { br ->
```

Comment:
please, do not use abbreviations in the project unless they are widely recognized. In this case "br" is quite confusing. Use "it" instead, or give it a more meaningful name.